x/concentrated-liquidity/bench_test.go

```diff
@@ -0,0 +1,235 @@
+			upperTick = upperTick - upperTick%tickSpacing
+
+			tokenDesired0 := sdk.NewCoin(denom0, sdk.NewInt(rand.Int63n(maxAmountDeposited)))
+			tokenDesired1 := sdk.NewCoin(denom1, sdk.NewInt(rand.Int63n(maxAmountDeposited)))
+
+			accountIndex := rand.Intn(len(s.TestAccs))
+
+			s.createPosition(accountIndex, poolId, tokenDesired0, tokenDesired1, lowerTick, upperTick)
+		}
+
+		// Setup numberOfPositions full range positions for deeper liquidity.
+		if shouldCreateFullRangePositions {
+			for i := 0; i < numberOfPositions; i++ {
+				lowerTick := types.MinTick
+				upperTick := types.MaxTick
+
+				tokenDesired0 := sdk.NewCoin(denom0, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokenDesired1 := sdk.NewCoin(denom1, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokensDesired := sdk.NewCoins(tokenDesired0, tokenDesired1)
+
+				accountIndex := rand.Intn(len(s.TestAccs))
+
+				account := s.TestAccs[accountIndex]
+
+				simapp.FundAccount(s.App.BankKeeper, s.Ctx, account, tokensDesired)
+
+				s.createPosition(accountIndex, poolId, tokenDesired0, tokenDesired1, lowerTick, upperTick)
+			}
+		}
+
+		// Setup numberOfPositions * 2 positions at random ranges around the current tick for deeper
+		// liquidity.
+		if shouldConcentrate {
+			// Within 10 ticks of the current
+			for i := 0; i < numberOfPositions; i++ {
+				lowerTick := currentTick.Int64() - 10
+				upperTick := currentTick.Int64() + 10
+
+				tokenDesired0 := sdk.NewCoin(denom0, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokenDesired1 := sdk.NewCoin(denom1, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokensDesired := sdk.NewCoins(tokenDesired0, tokenDesired1)
+
+				accountIndex := rand.Intn(len(s.TestAccs))
+
+				account := s.TestAccs[accountIndex]
+
+				simapp.FundAccount(s.App.BankKeeper, s.Ctx, account, tokensDesired)
+
+				s.createPosition(accountIndex, poolId, tokenDesired0, tokenDesired1, lowerTick, upperTick)
+			}
+
+			// Within 100 ticks of the current
+			for i := 0; i < numberOfPositions; i++ {
+				lowerTick := currentTick.Int64() - 100
+				upperTick := currentTick.Int64() + 100
+
+				tokenDesired0 := sdk.NewCoin(denom0, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokenDesired1 := sdk.NewCoin(denom1, sdk.NewInt(maxAmountDeposited).MulRaw(5))
+				tokensDesired := sdk.NewCoins(tokenDesired0, tokenDesired1)
+
+				accountIndex := rand.Intn(len(s.TestAccs))
+
+				account := s.TestAccs[accountIndex]
+
+				simapp.FundAccount(s.App.BankKeeper, s.Ctx, account, tokensDesired)
+
+				s.createPosition(accountIndex, poolId, tokenDesired0, tokenDesired1, lowerTick, upperTick)
+			}
+		}
+
+		swapAmountIn := sdk.MustNewDecFromStr(amountIn).TruncateInt()
+		largeSwapInCoin := sdk.NewCoin(denomIn, swapAmountIn)
+
+		liquidityNet, err := clKeeper.GetTickLiquidityNetInDirection(s.Ctx, pool.GetId(), largeSwapInCoin.Denom, currentTick, sdk.Int{})
+		noError(err)
+
+		fmt.Println("num_ticks_traversed", len(liquidityNet))
+		fmt.Println("current_tick", currentTick)
+
```

Comment:
nit: consider printing amount of total positions created

Reply:
It's configured by the user so this is assumed to be known when running the test